### PR TITLE
Improve descriptions for schemas w/o `name` and compound type-names

### PR DESCRIPTION
**Pull request by mmandlis**
https://github.com/PolymerLabs/arcs/issues/452

### CODE REVIEW

File: runtime/description.js [hunk 1]
```diff
@@ -225,16 +225,20 @@ class ValueToken {
     }
     if (view.type.isView) {
       let viewList = view.toList();
-      if (viewList) {
-        if (viewList.length > 2) {
-          // TODO: configurable view display format.
-          return `<b>${viewList[0].rawData.name}</b> plus <b>${viewList.length-1}</b> other items`;
+      if (viewList && viewList.length > 0) {
+        if (viewList[0].rawData.name) {
```
Comment (sjmiles) [inline runtime/description.js]: Why `rawData.name` and not just `name`?

Comment (mmandlis) [reply runtime/description.js]: entity has 2 data members - id and rawName, but rawName contains all the properties of the schema type.

Comment (sjmiles) [reply runtime/description.js]: Ok. I think I have PEC confusion. Is true to say that inner-PEC Entity instances have schema-aware getters/setters (and also rawData is a Proxy)?

Comment (mmandlis) [reply runtime/description.js]: right, on the outer PEC side the entity's rawData is just JSON.

Comment (sjmiles) [reply runtime/description.js]: Thank you!

**Pull request merged**